showcase/app/templates/page-components/advanced-table.hbs

```diff
@@ -2073,12 +2256,14 @@
                       {{capitalize state}}
                     </span>
                   </div>
-                  <Hds::AdvancedTable::ThResizeHandle
-                    mock-state-value={{state}}
-                    @column={{this.thResizeHandleColumn}}
-                    @nextColumn={{@nextColumn}}
-                    @tableHeight={{@tableHeight}}
-                  />
+                  {{#if this.thResizeHandleTable}}
```

Comment:
[question] is this needed? I tried to remove it locally, and Glint didn't complain; I've run `pnpm lint` and it passed.

Reply:
Ah yes it is not needed anymore, in an earlier iteration it could have been undefined.